pandas/core/arrays/integer.py

```diff
@@ -268,6 +265,13 @@ def _from_sequence(cls, scalars, dtype=None, copy=False):
     def _from_factorized(cls, values, original):
         return integer_array(values, dtype=original.dtype)
 
+    def _formatter(self, boxed=False):
+        def fmt(x):
+            if isna(x):
+                return 'NaN'
```

Comment:
should `NaN` have been hardcoded here?

Reply:
Why not? This is used for the Integer data display, where we currently use NaN.

(whether we should use rather 'NA' instead of 'NaN', that's another question)

Reply:
just wondering whether it would be a problem with `to_string(na_rep=...)`. will do some tests.

Reply:
Ah, yes, that's a good reason. But in general, this `_formatter` does not follow display options at all, is that correct? 
In which case this is something to think about in general.